sdk/storage/azure-storage-files-shares/CMakeLists.txt

```
@@ -14,7 +14,7 @@ set (AZURE_STORAGE_SHARES_HEADER
     inc/azure/storage/files/shares/share_responses.hpp
     inc/azure/storage/files/shares/share_sas_builder.hpp
     inc/azure/storage/files/shares/share_service_client.hpp
-    inc/azure/storage/files/shares/shares.hpp
+    inc/azure/storage/shares.hpp
```

Same here, keep the `/files/` subdirectory:
```suggestion
    inc/azure/storage/files/shares.hpp
```